src/Cake.FileHelpers/Cake.FileHelpers.csproj

```
@@ -1,7 +1,6 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net50;netstandard2.0;net46</TargetFrameworks>
-    <NetStandardImplicitPackageVersion Condition=" '$(TargetFramework)' == 'netstandard1.6' ">1.6.1</NetStandardImplicitPackageVersion>
+    <TargetFrameworks>netstandard2.0;net461;netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
```

```suggestion
    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
```

Both, `net461` and `netstandard2.0` are no longer a requirement, as of https://cakebuild.net/docs/extending/addins/best-practices#net-target-version

---

done